### PR TITLE
Airport read user logged in

### PR DESCRIPTION
**Pull request by BaileyEDennis**
## Description
Added feature where only logged in users see buttons. 

## Related Issue
#20 

## Motivation and Context
This is required so that non-authorized users cannot mess with the data on the site. 

## How Can This Be Tested?
git fetch airportReadUserBD

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/javascripts/helpers/viewHelper.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import home from '../components/views/homeView';
 import airport from '../components/views/airportView';
 
-const viewHelper = (id) => {
-  $('#app').html('');
+const viewHelper = (id, user) => {
   switch (id) {
+    case '#':
```
Comment (drteresavasquez) [inline src/javascripts/helpers/viewHelper.js:6]: For this, to simplify your code, you could do this as well:
```
case '#':
case 'home':
      $('#app').html('');
      return home.showHomepage();
```

Just an FYI

**Comment by BaileyEDennis** [comment]
Thanks! I will go implement that!

**Review by AutumnFoto — APPROVED** [review]
awesome!

**Review by wmmcinturff — APPROVED** [review]
Fetched to my local and checked; asked question about login. Approved.

**Pull request merged**